cookbook/recipes/markets/ingredients/templates/skeleton/app/components/Link.tsx

```diff
@@ -0,0 +1,12 @@
+import {Link as RemixLink} from '@remix-run/react';
+import {RemixLinkProps} from '@remix-run/react/dist/components';
+import {useSelectedLocale} from '../lib/i18n';
+
+export function Link({...props}: RemixLinkProps) {
+  const selectedLocale = useSelectedLocale();
+  const to =
+    selectedLocale != null
+      ? `${selectedLocale.pathPrefix}${props.to}`
+      : props.to;
```

Comment:
The `i18n` from the loader looks to always return a selected locale, at least a default one. This means that if I ended up on `mysite.com/` and the default locale is `EN-US`, a link to a product would be `mysite.com/en-us/products/handle`. If on the default locale, shouldn't it not prepend the locale?

Reply:
Maybe define the default locale as a const and export it also from `i18n.ts`, so we can test for it here?